### PR TITLE
Improvements: Tests

### PR DESCRIPTION
**Pull request by JuiP**
#41 In Progress

### CODE REVIEW

**Comment by JuiP** [comment]
I find one more thing very confusing... Didn't notice this before but the 'estimator' and 'Estimator' variables are very inconsistent. I guess I should make them consistent throughout the test files and the Helper functions?

**Comment by ataymano** [comment]
Looks like all changes from here are already included #43, so we can cancel this one?

File: estimators/test/test_bandits.py [hunk 1]
```diff
@@ -10,39 +10,41 @@
 from estimators.bandits import clopper_pearson
 from estimators.test.utils import Helper
 
-def test_bandits_unit_test():
-    listofestimators = [(ips.Estimator(), 2.0), (snips.Estimator(), 1.0), (mle.Estimator(), 1.0), (cressieread.Estimator(), 1.0)]
-    
+random.seed(0)
```
Comment (ataymano) [inline estimators/test/test_bandits.py:13]: can you please use fixtures for that?
https://docs.pytest.org/en/latest/how-to/fixtures.html
also see first comment here:
https://github.com/pytest-dev/pytest/issues/667

**Comment by JuiP** [comment]
@ataymano Originally, the plan was to keep work on items from the project board separate from improvements. But right now, yes it makes sense to close this PR.

**Pull request closed without merging**